test/syntax.jl

```diff
@@ -3746,6 +3746,15 @@ end
     let; execs += 1; array; end::Vector{Int} .= 7
     @test array == [7]
     @test execs == 4
+
+    # remove argument side effects on lhs kwcall
+    pa_execs = 0
+    kw_execs = 0
+    f60152(v, pa; kw) = copy(v)
+    @test (f60152([1, 2, 3], 0; kw=0) .*= 2) == [2,4,6]
```

Comment:
Should this function do something with the kw, to make sure it actually as the right value?